cli/dcoscli/data/help/node.txt

```diff
@@ -48,6 +51,9 @@ Options:
     --filter=<filter>
         Filter logs by field and value. Filter must be a string separated by colon.
         For example: --filter _PID:0 --filter _UID:1.
+        OR
+        Filter metrics by field. Filter must be a valid metrics field.
+        For example: --filter memory.total --filter memory.swap
```

Comment:
Let see how it looks? A few lines is fine.

I'm a little confused by the name `--filter`. `dcos node metric` shows a "summary", so when we "filter" we aren't really filtering but showing more information that's otherwise hidden, right?